CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Removed
 
+- Removed Trainer argument `profiler` as bool ([#6164](https://github.com/PyTorchLightning/pytorch-lightning/pull/6164))
```

Comment:
```suggestion
- Removed support for passing a bool value to `profiler` argument of Trainer ([#6164](https://github.com/PyTorchLightning/pytorch-lightning/pull/6164))
```
I suggest this reformulation to be clear we are removing the bool support, not the argument.